### PR TITLE
spring servlet example.

### PR DESCRIPTION
**Pull request by rghetia**

### CODE REVIEW

File: examples/spring/servlet/src/main/java/io/opencensus/examples/spring/servlet/Application.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    HttpViews.registerAllServerViews();
+    HttpViews.registerAllClientViews();
+
+    // Register Prometheus exporters and export metrics to a Prometheus HTTPServer.
+    // Refer to https://prometheus.io/ to run Prometheus Server.
+    PrometheusStatsCollector.createAndRegister();
+    HTTPServer prometheusServer = new HTTPServer(9090, true);
+  }
+
+  private static void initTracing() {
```
Comment (dinooliva) [inline examples/spring/servlet/src/main/java/io/opencensus/examples/spring/servlet/Application.java]: nit - consider naming something analogous to stats init - e.g. initTracingAndLoggingExporter

File: examples/spring/servlet/src/main/java/io/opencensus/examples/spring/servlet/HelloController.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  /**
+   * Serves index page.
+   *
+   * @return String
+   */
+  @RequestMapping("/")
+  public String index() {
+    String str = "Hello from servlet instrumented with opencensus-spring";
+    String resp = restTemplate.getForObject("http://localhost:8080/loopback", String.class);
+
```
Comment (dinooliva) [inline examples/spring/servlet/src/main/java/io/opencensus/examples/spring/servlet/HelloController.java]: nit - should the port be configurable?

Comment (rghetia) [reply examples/spring/servlet/src/main/java/io/opencensus/examples/spring/servlet/HelloController.java]: done.

**Comment by rghetia** [comment]
> LGTM overall. One nit: it would be great to exclude the gradle wrappers under examples if there's a way to do so.

I tried but have not found a good way to do. Earlier I had created sub-project under example/settings.gradle. That did not work because CreateStartScripts did not for work for subproject. May be there is a way.

**Pull request merged**